lib/src/showcase.dart

```diff
@@ -468,7 +473,7 @@ class _ShowcaseState extends State<Showcase> {
     blur = kIsWeb && blur < 0 ? 0 : blur;
 
     if (!_showShowCase) return const Offstage();
-
+    widget.onTargetWidgetSize?.call(size);
```

Comment:
The size you are using is screen size from `MediaQuery` not target widget size.

Reply:
I guess the screen size is named `screenSize`.

Reply:
Ok but this will get called every time widget updates.